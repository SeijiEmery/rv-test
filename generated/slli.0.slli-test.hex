0000000    13  11  11  00  93  91  11  00  13  12  12  00  13  00  00  00
0000020    13  00  00  00  13  00  00  00  13  00  00  00  13  00  00  00
0000040    73  00  10  00                                                
0000044
