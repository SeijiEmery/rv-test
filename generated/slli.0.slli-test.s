slli x2, x2, 1
slli x3, x3, 1
slli x4, x4, 1
nop
nop
nop
nop
nop
ebreak
