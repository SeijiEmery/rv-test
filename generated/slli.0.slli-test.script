load /x 0 generated/slli.0.slli-test.hex
writereg 1 1
writereg 2 2
writereg 3 0
writereg 4 31
setpc 0
run 100000
readreg 1
readreg 2
readreg 3
readreg 4