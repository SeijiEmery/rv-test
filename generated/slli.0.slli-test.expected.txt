R1 = 1
R2 = 4
R3 = 0
R4 = 62
